tcms/testruns/static/testruns/js/get.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(() => {
+
+    $('.bootstrap-switch').bootstrapSwitch();
+
+    const testRunId = $('#test_run_pk').data('pk')
+
+    $('.set-running-button').on('click', () => {
+        // use the JSON RPC API to set the date to null
+        // TODO: how - null, undefined, '0' not working!
+    })
+    
+    $('.set-finished-button').on('click', () => {
+        const now = new Date()
+
+        // TODO: better way to do this
+        const data = {'stop_date': `${now.getFullYear()}-${now.getMonth()+1}-${now.getDate()}`}
+        jsonRPC('TestRun.update', [testRunId, data], () => location.reload(true))
```

Comment:
This button doesn't work for me at all. I don't see JSON RPC request sent, the page doesn't reload, stop_date isn't updated. Have you tested it ?

Reply:
I have tested it, and it is NOT working. this is leftover code from the previous buttons. I removed it.

For the new switch button new code should be written, but I can do that in a separate PR once this is merged